tmk_core/protocol/report.h

```diff
@@ -207,8 +208,23 @@ typedef int16_t mouse_xy_report_t;
 typedef int8_t mouse_xy_report_t;
 #endif
 
+#ifdef MOUSE_SCROLL_EXTENDED_REPORT
+typedef int16_t mouse_hv_report_t;
+#else
+typedef int8_t mouse_hv_report_t;
+#endif
+
+#ifdef MOUSE_SCROLL_HIRES_ENABLE
+#    ifndef MOUSE_SCROLL_MULTIPLIER
+#        define MOUSE_SCROLL_MULTIPLIER 120
+#    elif (MOUSE_SCROLL_MULTIPLIER >= 120 || MOUSE_SCROLL_MULTIPLIER < 1)
```

Comment:
This checks of `MOUSE_SCROLL_MULTIPLIER` check if it's greater or *equal to* 120, but the default value is 120 and the error message says that 120 is a valid value.

Reply:
Yeah that is a mistake on my part thanks for catching that will put it in the next update.